libedk2_tpm/Tpm2Capability.c

```diff
@@ -566,6 +566,7 @@ Tpm2TestParms (
     default:
       return EFI_INVALID_PARAMETER;
     }
+    break;
```

Comment:
Code is the same as https://github.com/tianocore/edk2/blob/master/SecurityPkg/Library/Tpm2CommandLib/Tpm2Capability.c
Could you double confirm？